### PR TITLE
TSWEB-150

### PR DESCRIPTION
**Pull request by AlonaTechStack**

### CODE REVIEW

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/ApplyTitlePage.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace PlaywrightAutomation.Pages.AboutUs
+{
+    public class ApplyTitlePage : BasePage, IWebContainer
+    {
+        public string Container => "//div[@id='apply-title']";
```
Comment (DaniilVostrikov) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/ApplyTitlePage.cs:5]: Try to make it like component and others 'section' elements below

**Review by DaniilVostrikov — CHANGES_REQUESTED** [review]

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/EngineeringCulturePage.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightAutomation.Pages.AboutUs
+{
+    internal class EngineeringCulturePage : BasePage, IWebContainer
+    {
+        public string Container => "//div[@id ='engineeringCulture']";
+        public ILocator BlockTitle => Page.Locator(Container).Locator("//div[@class='block-title']");
```
Comment (anastasiia-liashova) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/EngineeringCulturePage.cs:8]: maybe "//div[contains(@class,'block-title')]"

Comment (anastasiia-liashova) [reply ui_tests/PlaywrightAutomation/Pages/AboutUs/EngineeringCulturePage.cs:8]: github perceive class as user))))

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/EngineeringCulturePage.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        public ILocator TextDescription => Page.Locator(Container).Locator("//div[@class='text-description']");
```
Comment (anastasiia-liashova) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/EngineeringCulturePage.cs:9]: same

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/OurAchievementsPage.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightAutomation.Pages.AboutUs
+{
+    internal class OurAchievementsPage : BasePage, IWebContainer
+    {
+        public string Container => "//div[@id='ourAchievements']";
+        public ILocator Achievements => Page.Locator(Container).Locator("//div[contains(@class,'AchievementInfoBlock')]");
+        public ILocator Title => Achievements.Locator("//h2");
```
Comment (anastasiia-liashova) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/OurAchievementsPage.cs:9]: maybe make a more specific xpath

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/TechstackRolesPage.cs [hunk 5]
```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightAutomation.Pages.AboutUs
+{
+    internal class TechstackRolesPage : BasePage, IWebContainer
+    {
+        public string Container => "//div[@id='techstackRoles']";
+        public ILocator ParagraphDescription => Page.Locator(Container).Locator("//div[@class='paragraph-description']");
```
Comment (anastasiia-liashova) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/TechstackRolesPage.cs:8]: contains class

File: ui_tests/PlaywrightAutomation/Pages/AboutUs/AboutUsPage.cs [hunk 6]
```diff
@@ -0,0 +1,17 @@
+﻿namespace PlaywrightAutomation.Pages.AboutUs
+{
+    internal class AboutUsPage : BasePage, IWebContainer
+    {
+        public string Container => "//div[contains(@class,'MainLayoutWrapper')]";
+        public string Title => "//div[@class='paragraph-title']";
```
Comment (anastasiia-liashova) [inline ui_tests/PlaywrightAutomation/Pages/AboutUs/AboutUsPage.cs:6]: it is better to use **contains**(class,....) in all of the xpath

**Pull request closed without merging**